Eatery/Views/Eateries/EateryExceptionsView.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  EateryExceptionsView.swift
+//  Eatery
+//
+//  Created by William Ma on 9/25/20.
+//  Copyright © 2020 CUAppDev. All rights reserved.
+//
+
+import UIKit
+
+class EateryExceptionsView: UIView {
+
+    private let blurView = UIVisualEffectView(effect: UIBlurEffect(style: .light))
+
+    private let imageView = UIImageView()
+
+    private let label = UILabel()
+
+    init() {
+        super.init(frame: .zero)
+
+        blurView.alpha = 0.5
+        blurView.clipsToBounds = true
+        addSubview(blurView)
+        blurView.snp.makeConstraints { make in
+            make.edges.equalToSuperview()
+        }
+
+        imageView.contentMode = .scaleAspectFit
+        imageView.tintColor = .white
+        insertSubview(imageView, aboveSubview: blurView)
+        imageView.snp.makeConstraints { make in
+            make.leading.equalToSuperview().inset(8)
+            make.top.bottom.equalToSuperview().inset(4)
+            make.width.height.equalTo(16)
+        }
+
+        label.font = .systemFont(ofSize: 12)
+        label.textColor = .white
+        insertSubview(label, aboveSubview: blurView)
+        label.snp.makeConstraints { make in
+            make.leading.equalTo(imageView.snp.trailing).offset(4)
+            make.top.bottom.equalToSuperview().inset(4)
+            make.trailing.equalToSuperview().inset(8)
```

Comment:
don't know what the convention has been in eatery thus far, but with the repeat of the constant padding values I could see a case for making constants and naming them so they describe what they are -- but really just a nitpicky thing. I'd just stick to whatever matches the current codebase does

Reply:
Current codebase uses literals